experimental/dds/tree2/docs/Shared Tree Philosophy.md

```diff
@@ -0,0 +1,194 @@
+# Shared Tree
+
+The goal of the shared tree project is empower developers to create and maintain better collaborative experiences.
+
+To best achieve this goal we plan to deliver a set of libraries that enable developers unfamiliar with the challenges specific to collaborative applications to:
+
+-   Adopt Shared Tree without concern for having to migrate to another solution due to hitting limitations.
+
+    This means that if a developer wants to make a change that interacts with collaboration, the fact that their application is powered by shared tree will not be a limiting factor on what can be achieved.
+    This includes compatibility between different application or version of an applications, scalability, availability, merge quality etc.
+    More concretely, if faced with a new application collaboration requirement, it should be practical to either build it on top of shared tree or extend shared tree to support it.
+    Doing this should not require major reworking of either the application or shared tree implementations, not require extending it in a way that the upstream shared tree would not be willing to maintain.
+
+-   Productively create and maintain collaborative experiences.
```

Comment:
I think this should be the first point, since this is the actual problem Shared Tree is solving. The point about migration could be moved after the pitfalls bullet, since it's an example of a specific pitfall.

Reply:
+1

Reply:
Done